v4-cokapi/cokapi.js

```diff
@@ -37,7 +37,7 @@ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 var IS_DEBUG = false;
 
-var PRODUCTION_PORT = 3000;
+var PRODUCTION_PORT = 3001;
```

Comment:
Good point. Yes, this kind of things should be passed as an argument to the program. Why do we need to change the frontend? I'm pretty sure you can set a default value.